packages/grid/data-grid/src/tests/filtering.DataGrid.test.tsx

```diff
@@ -251,6 +251,110 @@ describe('<DataGrid /> - Filter', () => {
       expect(getColumnValues()).to.deep.equal(['Nike']);
     });
 
+    it('should allow operator isEmpty for number field', () => {
```

Comment:
There's a "numeric operators" group only for tests related to the operators available in the `number` column type.